product_docs/docs/edbcloud/beta/overview/03_security_compliance.mdx

```diff
@@ -0,0 +1,30 @@
+---
+title: "Security and Compliance"
+---
+
+### Security Governance
+
+- Cloud Security Compliance officer is responsible for maintaining compliance targets for their entire organization.
+- EDB Cloud infrastructure periodically undergoes SOC2 audit trials to ensure that EDB Cloud is always compliant with security requirements.
+- All EDB staff undergo background checks before they are hired.
+- All EDB staff are required to take mandatory security training.
+- The EDB Software Development Life Cycle integrates continuous security scans of added/updated code.
+
+
+### EDB Cloud Security Features
+EDB Cloud is a Database as a Service (DBaaS) which runs in your own Cloud account, isolating your data from other users & giving you control over our access to it. The key security features of EDB Cloud are as follows:
```

Comment:
We don't need to describe what EDB Cloud is here. But I think we should probably have a top-level or second-level header about running in your cloud account, and what the implies.

Reply:
@drsm79 I can' respond to this ^^^

Reply:
Regarding these two paragraphs:

> EDB Cloud is a Database as a Service (DBaaS) which runs in your own Cloud account, isolating your data from other users & giving you control over our access to it. The key security features of EDB Cloud are as follows:
> - **Segregation of IT environments:** Customers database clusters are installed and managed in their own cloud environment. Complete segregation of customer's data is ensured.

I could not change that myself because I think they are not mine.  The first para was given by @drsm79  For the second para, I remember it took about half of  a day of discussion on slack with @nstott  to select proper language for this. @ba-edb you are in a better position to see the subtleties of this bullet points, please feel free to edit the language as appropriate. Thanks.

Reply:
...this specific repo has liberal rules. it is OK to edit files right in github editor and make small commits right into edbcloud-beta branch. The recommended prefix for commit on the ticket is "docs(UPM-876):  ". But commit message policy is not enforced